tools/nl/embeddings/input/base/2024_q3.csv

```diff
@@ -0,0 +1,8 @@
+dcid,sentence
+Count_Person_InvolvedInCrash_Motorists,Number of motorists involved in crash
+Count_Person_InvolvedInCrash_MotorVehicleOccupant,Number of motor vehicle occupants involved in crash
+Count_Vehicle_InvolvedInCrash_InTransport,Number of vehicles in motion involved in crash
```

Comment:
I see, so "in transport" does not mean being transported from place A to B. 

How about just say "moving vehicles"?